packages/sdk/src/types/index.ts

```diff
@@ -254,14 +254,23 @@ export type SDKConfig = Readonly<
 >;
 
 /**
- * ModuleInitializer Type represents a function accepting module options
+ * ModuleInitializer Type represents a function accepting optional module options
  * as an argument and returning the actual module.
  */
 export type ModuleInitializer<
   InitializedModule extends Module,
   Options extends ModuleOptions
 > = (options?: Options) => InitializedModule;
 
+/**
+ * StrictModuleInitializer Type represents a function accepting mandatory module options
+ * as an argument and returning the actual module.
+ */
+export type StrictModuleInitializer<
```

Comment:
What is stric? How can I use non-strict? Why not ModuleInitializer, can't we use optional options?

>